package/upgrade/upgrade_manifests.sh

```diff
@@ -1256,6 +1256,80 @@ patch_local_cluster_details() {
   kubectl patch -n fleet-local cluster.provisioning local --subresource=status --type=merge --patch '{"status":{"fleetWorkspaceName": "fleet-local"}}'
 }
 
+upgrade_rbac() {
+
+  # only versions before v1.3.0 that upgrading to v1.3.0 need this patch
+  if [[ ! "${UPGRADE_PREVIOUS_VERSION%%-rc*}" < "v1.3.0" ]]; then
+    echo "Only versions before v1.3.0 need this patch."
```

Comment:
I think we should make this PR in the v1.2 branch. The users have to upgrade from v1.2.1 to v1.2.x first.